code/modules/antagonists/cult/cult_items.dm

```diff
@@ -546,24 +556,31 @@
 		totalcurses++
 		to_chat(user, span_danger("You shatter the orb! A dark essence spirals into the air, then disappears."))
 		playsound(user.loc, 'sound/effects/glassbr1.ogg', 50, TRUE)
-		var/static/list/curses
-		if(!curses)
-			curses = list("A fuel technician just slit his own throat and begged for death.",
-			"The shuttle's navigation programming was replaced by a file containing just two words: IT COMES.",
-			"The shuttle's custodian was found washing the windows with their own blood.",
-			"A shuttle engineer began screaming 'DEATH IS NOT THE END' and ripped out wires until an arc flash seared off her flesh.",
-			"A shuttle inspector started laughing madly over the radio and then threw herself into an engine turbine.",
-			"The shuttle dispatcher was found dead with bloody symbols carved into their flesh.",
-			"The shuttle's transponder is emitting the encoded message 'FEAR THE OLD BLOOD' in lieu of its assigned identification signal.")
-		var/message = pick_n_take(curses)
-		message += " The shuttle will be delayed by three minutes."
-		priority_announce("[message]", "System Failure", 'sound/misc/notice1.ogg')
+
+		var/curse_message = "Something has gone horrendously wrong..."
+		for(var/i in 1 to 3) // can't use pick_and_take() with json, so we just do 3 shots at pulling a message we haven't used yet, probably good enough
```

Comment:
you could make this work but its not really necesasry ig uess

Reply:
you'd just call `json_load(CULT_SHUTTLE_CURSE)`, save its curse_announce field somewhere, then pick'n'take that